projects/storefrontapp-e2e-cypress/cypress/helpers/my-company/models/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './MyCompanyConfig';
```

Comment:
I believe these files should be renamed to `my-company-config.ts` or `my-company.config.ts` and also `my-company-row-config.ts` or `my-company-row.config.ts`